book/src/zcash/technical-details.md

```diff
@@ -121,4 +121,4 @@ instead of publishing them on the blockchain.
 Another idea is to extend lightwalletd servers to support messaging, since
 wallets are all already connected to a server (not the same one, so inter-server
 communications would be also needed)
-```
\ No newline at end of file
+```
```

Comment:
Can you revert this line change? I don't see why it was removed and then added back in. Thank you!